Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ghcr.io/linuxserver/baseimage-alpine:3.13
+
+ARG BUILD_DATE
+ARG VERSION
+ARG ADGUARDHOMESYNC_RELEASE
+LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
+LABEL maintainer="TheSpad"
+
+ENV \
+  HOME=/app \
+  GO111MODULE=on \
+  GOOS=linux
+
+RUN \
+  apk add --update --no-cache curl && \
+  mkdir -p /app/adguardhome-sync && \
+  if [ -z ${ADGUARDHOMESYNC_RELEASE+x} ]; then \
+	  ADGUARDHOMESYNC_RELEASE=$(curl -sX GET "https://api.github.com/repos/bakito/adguardhome-sync/releases/latest" \
+	  | awk '/tag_name/{print $4;exit}' FS='[""]'); \
+  fi && \
+  echo "*** Installing AdGuardHome Sync ***" && \
+  curl -o \
+	  /tmp/adguardhomesync.tar.gz -L \
+    "https://github.com/bakito/adguardhome-sync/releases/download/${ADGUARDHOMESYNC_RELEASE}/adguardhome-sync_${ADGUARDHOMESYNC_RELEASE#v}_linux_x86_64.tar.gz" && \
+  tar xzf \
+	  /tmp/adguardhomesync.tar.gz -C \
+	  /app/adguardhome-sync/ && \
+  echo "*** Cleaning Up ***" && \
+  rm /tmp/adguardhomesync.tar.gz
```

Comment:
Internet said; tabs vs spaces?
Spad replies: why not both.

Reply:
Spad is very confused because that's 2 and 4 spaces in vscode

Reply:
Oh, it's because I copied iX's code from overseerr for those lines so not my fault. Fixing now.